File: developer/provenance/haathi/src/lib/com/haathi/utils/AuthenticationCredentials.cpp
/******************************************************************************
Header
******************************************************************************/
// $Header: //MyDataDepot/Projects/provenance-autoType2/haathi/src/lib/com/haathi/utils/AuthenticationCredentials.cpp#1 $
/******************************************************************************
Include self
******************************************************************************/
#include "com/haathi/utils/AuthenticationCredentials.hpp"
/******************************************************************************
Include others
******************************************************************************/

/******************************************************************************
Define
******************************************************************************/

/******************************************************************************
Using
******************************************************************************/

/******************************************************************************
Namespace
******************************************************************************/
namespace com {
	namespace haathi {
		namespace utils {
/******************************************************************************
Class
******************************************************************************/
AuthenticationCredentials::AuthenticationCredentials() {
}

void AuthenticationCredentials::setEmailAddress(OUString emailAddress) {
	this->emailAddress = emailAddress;
}

OUString AuthenticationCredentials::getEmailAddress() {
	return emailAddress;
}

void AuthenticationCredentials::setPassword(std::string password) {
	this->password = password;
}

std::string AuthenticationCredentials::getPassword() {
	return password;
}

AuthenticationCredentials authenticationCredentials;
/******************************************************************************
Namespace
******************************************************************************/
		};
	};
};
/*****************************************************************************/

File: developer/provenance/servers/TrustedServer/src/secure/provenance/AuthenticationCredentials.java
/* ***************************************************************************
 $Header: //MyDataDepot/Projects/provenance-autoType2/servers/TrustedServer/src/secure/provenance/AuthenticationCredentials.java#1 $
 *****************************************************************************
 Package
 ****************************************************************************/
package secure.provenance;
/* ***************************************************************************
 Imports
 ****************************************************************************/
import org.json.JSONObject;
/* ***************************************************************************
 Class
 ****************************************************************************/
public class AuthenticationCredentials {
	private String emailAddress;
	private String password;

	public AuthenticationCredentials(JSONObject jsonObject) {
		this(jsonObject.getString("emailAddress"), jsonObject.getString("password"));
	}
	
	public AuthenticationCredentials(String emailAddress, String password) {
		this.emailAddress = emailAddress;
		this.password = password;
	}

	public String getEmailAddress() {
		return emailAddress;
	}

	public String getPassword() {
		return password;
	}
}
/* **************************************************************************/
